Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/CovidTreatmentAssessment.vue

```diff
@@ -0,0 +1,488 @@
+                                    :has-additional-response="true"
+                                />
+                                <span class="error-message">
+                                    {{ errors[0] }}
+                                </span>
+                            </ValidationProvider>
+                        </Card>
+                        <Card
+                            title="7. Have you had 3 doses of the vaccine?"
+                            additional-info="Citizen has had 3 doses of vaccine for more than 14 days."
+                        >
+                            <ValidationProvider
+                                ref="reports3DosesC19Vaccine"
+                                v-slot="{ errors }"
+                                rules="oneOf:Yes,No,NotSure"
+                                v-bind="$attrs"
+                            >
+                                <OptionDetails
+                                    :value.sync="
+                                        covidTreatmentAssessmentRequest.reports3DosesC19Vaccine
+                                    "
+                                    :has-not-sure-option="true"
+                                    :has-additional-response="true"
+                                />
+                                <span class="error-message">
+                                    {{ errors[0] }}
+                                </span>
+                            </ValidationProvider>
+                        </Card>
+                        <Card
+                            title="8. Have you been diagnosed by a health care provider with a chronic condition?"
+                            additional-info="Citizen has a chronic condition."
+                        >
+                            <OptionDetails
+                                :value.sync="
+                                    covidTreatmentAssessmentRequest.hasChronicConditionDiagnoses
+                                "
+                                :has-not-sure-option="true"
+                                :has-additional-response="true"
+                            />
+                        </Card>
+                        <Card title="Notes">
+                            <div class="pt-2">
+                                <v-textarea
+                                    maxlength="2000"
+                                    counter="2000"
+                                    filled
+                                    auto-grow
+                                    rows="4"
+                                    :value="
+                                        covidTreatmentAssessmentRequest.agentComments
+                                    "
+                                />
+                            </div>
+                        </Card>
+                    </v-form>
+                </ValidationObserver>
+                <v-row class="py-3">
```

Comment:
We'll need to add a checkbox for modifying the address and a read-only version of the phone number entered above.